--- ports/stm32f4/linker/stm32f4_boot.ld
+++ ports/stm32f4/linker/stm32f4_boot.ld
@@ -43,7 +43,7 @@
 /* Specify the memory areas */
 MEMORY
 {
-  RAM (xrw)     : ORIGIN = 0x20000000, LENGTH = 64K - 4 /* reserve 4 bytes for double tap */
+  RAM (xrw)     : ORIGIN = 0x20000000, LENGTH = 128K - 4 /* reserve 4 bytes for double tap */
   FLASH (rx)    : ORIGIN = 0x08000000, LENGTH = 31K
   CONFIG (rx)   : ORIGIN = 0x08008000 - 1024, LENGTH = 1024
 }
